libraries/lib-numeric-formats/formatters/ParsedNumericConverterFormatter.cpp

```diff
@@ -170,7 +170,7 @@ class ParsedNumericConverterFormatter final :
    {
       UpdateFormat();
 
-      if (mType == NumericConverterType_TIME)
+      if (IsTimeRelatedFormat())
```

Comment:
I understand this substitution, but then I wanted to check that it was done completely.

I see an omission in `ParsedNumericConverterFormatterFactory`.